main.go

```diff
@@ -2,229 +2,62 @@ package main
 
```

Comment:
good refactoring, having main as small as possible is always a good idea.
could be improved by having a `cmd/kubenurse/cmd.go` file which does all the heavylifting of the instantion, and main.go that simply calls `Cmd.Run()`

seems to be a well-adopted convention: https://github.com/golang-standards/project-layout

Reply:
Yes, but since there are no other subcommands I'll leave it as it is now, I think there is no need to do this unless we have multiple subcommands.